src/test/java/com/uid2/admin/vertx/SharingServiceTest.java

```diff
@@ -559,37 +558,38 @@ void KeysetSetBadWhitelistSiteId(Vertx vertx, VertxTestContext testContext) {
         });
     }
 
-    @Test
-    void KeysetSetNewIdenticalNameAndSiteId(Vertx vertx, VertxTestContext testContext) {
-        fakeAuth(Role.ADMINISTRATOR);
-
-        mockSiteExistence(8, 22, 25, 6);
-
-        Map<Integer, Keyset> keysets = new HashMap<Integer, Keyset>() {{
-            put(1, new Keyset(1, 8, "test", Set.of(4,6,7), Instant.now().getEpochSecond(),true, true));
-        }};
-
-        setKeysets(keysets);
-
-        String body = "  {\n" +
-                "    \"allowed_sites\": [\n" +
-                "      22,\n" +
-                "      25,\n" +
-                "      6\n" +
-                "    ],\n" +
-                "    \"site_id\": 8, \n" +
-                "    \"name\": \"TEST\" \n" +
-                "  }";
-
-        post(vertx, "api/sharing/keyset", body, ar -> {
-            assertTrue(ar.succeeded());
-            HttpResponse response = ar.result();
-            assertEquals(400, response.statusCode());
-            assertEquals("Keyset with same site_id and name already exists", response.bodyAsJsonObject().getString("message"));
-
-            testContext.completeNow();
-        });
-    }
+    // This test should be uncommented out when multiple keysets is enabled
```

Comment:
will @Disabled work here instead of commenting it out?

Reply:
switched this to disabled